tests/test_utilities.py

```diff
@@ -116,3 +116,39 @@ def make_age_bins():
         names=["age_start", "age_end", "age_group_name"],
     )
     return pd.DataFrame(index=idx).reset_index()
+
+
+def simple_pop_structure() -> pd.DataFrame:
+    # Create simple population structure
+    age_idx = pd.MultiIndex.from_tuples(
+        [
+            (0.0, 25.0, "Young People"),
+            (25.0, 50.0, "Old People"),
+            (50.0, 75.0, "Ancient People"),
```

Comment:
Wow, these age group names are aggressive :P

Reply:
Yeah. I think I must really be ancient b/c I view 25 year olds as babies.